posthog/api/authentication.py

```diff
@@ -88,6 +90,17 @@ def create(self, validated_data: Dict[str, str]) -> Any:
         if not user:
             raise serializers.ValidationError("Invalid email or password.", code="invalid_credentials")
 
+        require_verification_feature = posthoganalytics.feature_enabled("require-email-verification", str(user.uuid))
```

Comment:
use distinct ids instead, the uuid doesn't map to a posthog person :)

Reply:
(and I just notice you asked me about this below, but I didn't _see_ the uuid part 🙈 )

Reply:
It's so that we can roll it out slowly and easily roll it back if there are any issues. Once we verify that it's good for everyone we will remove the flag.